src/Sindarin/SindarinDebugger.class.st

```diff
@@ -144,9 +143,7 @@ SindarinDebugger >> jumpIntoBlock: aBlockNode toNode: targetNode [
 SindarinDebugger >> message: aSelector [
 	"Returns whether the execution is about to send a message of selector @aSelector to any object"
 
-	[ ^ self messageSelector = aSelector ]
-		on: Error
-		do: [ ^ false ]
+	^ self isMessageSend and: [ self messageSelector = aSelector ]
```

Comment:
Ok, but the method name is weird, I don't expect it to return a boolean.

It should be renamed: `isMessageSend:`, or something like that

Reply:
Same my comment disappeared.
I said this is part of the original code and a deeper refactoring should be done in a new PR.